src/js/background/assignManager.js

```diff
@@ -244,15 +244,18 @@ const assignManager = {
   },
 
   async getPermissions() {
```

Comment:
Rename `getPermissions` so that its purpose is more obvious. E.g. to `resetBookmarksMenuItem`.

Reply:
and this